apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/http/UrlEncode.kt

```diff
@@ -15,11 +15,11 @@ internal fun String.urlEncode(
     when {
       it in RESERVED_CHARS -> append(it.percentEncode())
       spaceToPlus && it == ' ' -> append('+')
-      else -> append(it.percentEncode())
+      else -> append(it)
```

Comment:
Yikes, thanks for catching this!

Reply:
No problem. I was in doubt to put it in this PR directly, but felt to small enough